### ISSUE TITLE
app produces invalid YAML

### ISSUE THREAD

**Issue by biowaffeln** [kbravh/tweet-to-markdown#3]
Hey! I noticed that the frontmatter that is produced by the app is actually invalid, because @ is a special character in YAML and text that starts with an @ must be quoted ([see here](https://cloudslang-docs.readthedocs.io/en/v1.0/yaml_overview.html#scalars-strings-numbers-values)).

For example:

```md
---
author: Josh ✨
handle: @JoshWComeau <-- should be "@JoshWComeau"
---
```

**Comment by kbravh** [comment]
That's great to know! Let me get a fix out for this.

**Comment by kbravh** [comment]
@biowaffeln I've published version [1.4.2](https://www.npmjs.com/package/tweet-to-markdown) to resolve this issue. Thanks for the feedback!
